array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(num => num*2);
+console.log('numbers: ', doubled);
+
+const prices = numbers.map(num => `$${num}.00`);
```

Comment:
While this works for this exercise, in the future if you want a number to have a specific amount of decimal places it is best to use `toFixed`.

```JAVASCRIPT
num => `$${num.toFixed(2)}`
```